### ISSUE TITLE
Add option to skip GitHub action based on commit message

### ISSUE THREAD

**Issue by Bisaloo** [ropensci/codemetar#282]
See https://github.com/ropensci/codemetar/pull/281#discussion_r367397287

**Comment by maelle** [comment]
For inspiration https://github.com/marketplace/actions/skip-based-on-commit-message

**Comment by maelle** [comment]
Aaah even better https://github.com/marketplace/actions/skip-commit

```
if: "!contains(github.event.head_commit.message, 'ci-skip')"
```

@Bisaloo would you like to make a PR for this btw (in the workflow used in dev, and in the workflow template that's in dev)? Only if you're interested.

**Comment by maelle** [comment]
The skip message could be "cm-skip" maybe.

**Comment by Bisaloo** [comment]
A possible issue I see here (but I would need to run some tests to check if that's actually the case) is the fact that the test runs on `head_commit.message`.

So if I push 3 commits at once:

- 'Change stuff'
- 'Change more stuff'
- 'Change even more stuff [cm-skip]'

From what I understand, the GitHub action won't run at all because the `HEAD` commit message contains `cm-skip`, even when I would like it to run for the first 2 commits.

But that's similar to what `ci-skip` does for travis so maybe that's not an issue. What do you think?

**Comment by maelle** [comment]
Ah, I think it's fine?

**Comment by maelle** [comment]
fixed in #283
